src/otx/algo/anomaly/openvino_model.py

```diff
@@ -190,7 +190,7 @@ def check_if_quantized(model: openvino.Model) -> bool:
             **ptq_config,
         )
 
-        openvino.save_model(compressed_model, output_model_path)
+        openvino.save_model(compressed_model, output_model_path, compress_to_fp16=False)
```

Comment:
It looks like this flag doesn't work in case of quantized model: only initial .save_model on export matters

Reply:
See https://github.com/openvinotoolkit/openvino/pull/23609